show_client/vlan_cli.go

```diff
@@ -0,0 +1,193 @@
+package show_client
+
+import (
+	"encoding/json"
+	"net"
+	"sort"
+	"strings"
+
+	log "github.com/golang/glog"
+	sdc "github.com/sonic-net/sonic-gnmi/sonic_data_client"
+)
+
+const VlanTable = "VLAN"
+const VlanInterfaceTable = "VLAN_INTERFACE"
+const VlanMemberTable = "VLAN_MEMBER"
+
+type VlanConfig struct {
+	VlanData      map[string]interface{}
+	VlanIpData    map[string]interface{}
+	VlanPortsData map[string]interface{}
+}
+
+type VlanBriefColumn struct {
+	Name   string
+	Getter func(cfg VlanConfig, vlan string) []string
+}
+
+var VlanBriefColumns = []VlanBriefColumn{
+	{"VLAN ID", getVlanId},
+	{"IP Address", getVlanIpAddress},
+	{"Ports", getVlanPorts},
+	{"Port Tagging", getVlanPortsTagging},
+	{"Proxy ARP", getProxyArp},
+}
+
+func getSortedKeys(m map[string]interface{}) []string {
```

Comment:
can you check if there is an existing helper function for this in show_common.go otherwise move this to show_common.go